app/items/equipment/eqipment.py

```diff
@@ -0,0 +1,14 @@
+from app.items.item import BaseItem
+
+
+class BaseEquipment(BaseItem):
+    def __init__(
```

Comment:
I would disagree with this design. The only thing that equipment is sharing is `name and is_equipped` attribute. With this every piece of equipment will have the same attrs which wouldnt make sense like hp/protection for  a sword.